src/app/tests/TestEventLogging.cpp

```diff
@@ -150,16 +150,18 @@ static void CheckLogReadOut(nlTestSuite * apSuite, chip::app::EventManagement &
     chip::TLV::TLVReader reader;
     chip::TLV::TLVWriter writer;
     uint8_t backingStore[1024];
-    size_t elementCount;
```

Comment:
I'd like to see a comprehensive list of unit tests that rigorously test the various tenets of the spec.

E.g

1. Prove that if you subscribe to a set of types, that the events you get when subscribing/reading are only that set, and in increasing event number.
2. Variations of the paths in the subscribe, from wild-card to specific types, and ensuring the stream you get is specific to that path set.
3. That if you were to provide a particular event number in the sub, that you get back events that and newer
4. Over-write policy validations, including ensuring that emission of a lower priority event doesn't over-write a higher priority event.
etc.

Reply:
FYI there are more test cases that the above set :)